.gitlab-ci.yml

```diff
@@ -143,6 +145,9 @@ variables:
 .if_not_test_kitchen_deploy: &if_not_test_kitchen_deploy
   if: $CI_COMMIT_BRANCH != "master" && $CI_COMMIT_TAG == null && $DEPLOY_AGENT != "true" && $CI_PIPELINE_SOURCE != "web" && $CI_PIPELINE_SOURCE != "api"
 
+.if_test_kitchen_security_agent: &if_test_kitchen_security_agent
```

Comment:
Maybe `if_agent_security_label` would be more descriptive.

Reply:
It is not used any more (I wanted to trigger the kitchen tests based on labels on the PR but they are not available). Removed